### DISCUSSION TITLE
Is it possible to set custom DNS on the Redisson open sourced version?

### DISCUSSION THREAD

**Discussion by maranza** [redisson/redisson#6228] [Q&A] [answered]
I am trying to connect to my redis cluster that is on AWS VPC however i notice that the resisson clients defaults to public google dns server, which then fails to resolve my aws vpc domain names, is it possible to manually set the dns server on the open sourced version?

Thanks in advance.

**Comment by mrniko** [comment]
Netty uses a system dns. Did you try to change it?

**Comment by mrniko** [comment, marked as answer]
As option you can override dns resolver through `addressResolverGroupFactory` setting
